x/staking/keeper/delegation.go

```diff
@@ -660,14 +660,16 @@ func (k Keeper) Undelegate(
 	return completionTime, nil
 }
 
-// CompleteUnbonding completes the unbonding of all mature entries in the
-// retrieved unbonding delegation object.
-func (k Keeper) CompleteUnbonding(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) error {
+// CompleteUnbondingWithAmount completes the unbonding of all mature entries in
+// the retrieved unbonding delegation object and returns the total unbonding
+// balance or an error upon failure.
+func (k Keeper) CompleteUnbondingWithAmount(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) (sdk.Coins, error) {
```

Comment:
Introducing a new function instead of replacing the old as we want to get this into the next 0.37 and 0.38 point releases.